### ISSUE TITLE
Mentor Schedule

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/nazareth#2]
https://docs.google.com/spreadsheets/d/1ibF-AWkiQJndBlONh1fnzlkMkzpCuyh1Fyksp9XFXuc/edit#gid=0

**Comment by Jbarget** [comment]
@rebeccaradding im not sure if the spreadsheet is up to date however one thing I am wary of is wasting resources by oversubscription for particular weeks.

For week 1 of nazareth there are 15 people signed up. I think its great to be encouraging people to come out to mentor but I just wanted to make sure that as people start to book that theres a clear strategy in place to avoid creating "too many cooks spoil the broth" situation.

To channel the community's enthusiasm i feel we should have some rules around when people can come out.

![screen shot 2016-12-14 at 16 52 56](https://cloud.githubusercontent.com/assets/11725595/21191686/e6ba9808-c21d-11e6-9d04-fe7f409dfb31.png)

**Comment by shireenzoaby** [comment]
I agree with @Jbarget, I think we should have 2 mentors per week, the problem is that F&C9 haven't updated the calendar and have assigned themselves to dates where they won't be available

**Comment by des-des** [comment]
## Update

Fac 9 will stay in London for first four weeks of FAC10/FACN0 (feb 20th - march 17th)
